msteams-platform/tabs/how-to/using-fluid-msteam.md

```diff
@@ -0,0 +1,317 @@
+---
+title: Using Fluid with Microsoft Teams
+author: timtwang
+ms.author: mobajemu
+description: Tutorial for integrating Fluid-powered real-time collaboration features into a Microsoft Teams tab application
+ms.localizationpriority: medium
+ms.topic: conceptual
+---
+
+# Use Fluid with Microsoft Teams
+
+This tutorial helps to integrate Fluid-powered real-time collaboration features into a [Microsoft Teams tab application](/microsoftteams/platform/tabs/what-are-tabs). By the end of this tutorial you'll be able to integrate any Fluid-powered applications into Microsoft Teams and collaborate with others in real-time.
+
+In this section you'll learn the following concepts:
+
+1. How to integrate a Fluid client into a Microsoft Teams tab application.
+2. How to run and connect your Teams application to a Fluid service (Azure Fluid Relay).
+3. How to create and get Fluid Containers and pass them to a React component.
+
+For an example of how this recipe may be used to build a more complex application, check out the [Teams Fluid Hello World](https://github.com/microsoft/FluidExamples/tree/main/teams-fluid-hello-world) example in our FluidExamples repo.
+
+> [!NOTE]
+> This tutorial assumes that you are familiar with the [Fluid Framework Overview](https://fluidframework.com/docs/) and that you have completed the [QuickStart](https://fluidframework.com/docs/start/quick-start/). You should also be familiar with the basics of [React](https://reactjs.org/), [React Hooks](https://reactjs.org/docs/hooks-intro.html), and [Microsoft Teams Tabs](/microsoftteams/platform/tabs/what-are-tabs).
```

Comment:
Since this content is no longer in the Fluid docset, I suggest we convert this note as a new "Prerequisites" section to the doc. The note "assumes" a familiarity with the intro articles of the Fluid docset, but readers dropping into this article from web search won't necessarily have that context. Listing the articles here in a prerequisite section would be better customer experience, I think.